fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/UpdatePropertiesService.java

```diff
@@ -40,4 +40,5 @@
   void updateProperties(final FedoraResource fedoraResource,
                         final String sparqlUpdateStatement,
                         final RdfStream originalTriples) throws MalformedRdfException, AccessDeniedException;
+
```

Comment:
should probably revert this since there aren't any actual changes to this file

Reply:
This has been fixed.